src/shipit_pulse_listener/tests/test_code_coverage_listener.py

```diff
@@ -83,6 +83,15 @@ def test_is_old_task():
             }]
         }
     }
+    assert not hook.is_old_task(old_task)
+
+    old_task = {
+        'status': {
+            'runs': [{
+                'resolved': (datetime.utcnow() - timedelta(4)).strftime('%Y-%m-%d'),
```

Comment:
Not sure if it makes sense, but since `runs` is an array, and your algo seems to detect "tasks that have at least one run older than 3 days", you could add another run before this one with:

    'resolved': datetime.utcnow().strftim('%Y-%m-%d')